api/cases/templates/case/index.js

```diff
@@ -0,0 +1,11 @@
+const divorce = require('./divorce.template.json');
+const sscs = require('./sscs.template.json');
+
+const templates = {
+    divorce: divorce,
+    sscs, sscs
```

Comment:
typo? sscs: sscs

this works though ... as mentioned above es6 allows for `divorce: divorce` to be simplified to `divorce` 

please remove last `sscs` and update line 5 to `divorce` only